flink-yarn/src/main/java/org/apache/flink/yarn/YarnClusterDescriptor.java

```diff
@@ -135,6 +138,8 @@
 	/** Lazily initialized list of files to ship. */
 	private final List<File> shipFiles = new LinkedList<>();
 
+	private final List<Path> sharedLibs = new LinkedList<>();
```

Comment:
If we want to test this feature like ship files, we may expose `getter/setter` methods of this field.